modules/vault/main.tf

```diff
@@ -43,9 +43,9 @@ resource "aws_s3_bucket" "vault-unseal-bucket" {
 // ----------------------------------------------------------------------------
 resource "aws_dynamodb_table" "vault-dynamodb-table" {
   name           = "vault-unseal-${var.cluster_name}-${local.vault_seed}"
-  billing_mode   = "PROVISIONED"
-  read_capacity  = 2
-  write_capacity = 2
+  billing_mode   = "PAY_PER_REQUEST"
```

Comment:
I would make them all variables instead of passing the values.

Reply:
do we agree that we use Free Tier On-Demand 25 RCU/WCU as default and provide option to use Provision and specify RCU and WCU?